ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/ovn/hypershift/guest-openldap-idp/cucushift-installer-rehearse-aws-ipi-ovn-hypershift-guest-openldap-idp-workflow.yaml

```diff
@@ -0,0 +1,12 @@
+workflow:
+  as: cucushift-installer-rehearse-aws-ipi-ovn-hypershift-guest-openldap-idp
```

Comment:
@dpuniaredhat , submitted https://github.com/openshift/release/pull/53680 so you can use env var in job config to avoid defining new workflow for each new IDP type.

Reply:
Ack